### ISSUE TITLE
Password reset error

### ISSUE THREAD

**Issue by Natalieprojectisizwe** [project-isizwe/wifi-chat#25]
Attempted get a reset password email sent to me got an error saying it failed.

**Comment by imaginator** [comment]
I think this is related to:

```
TypeError: Cannot read property 'user' of undefined
    at Xmpp.getJidType (/opt/wifi-chat/node_modules/xmpp-ftw/lib/xmpp.js:270:32)
    at Sparky.<anonymous> (/opt/wifi-chat/index.js:138:29)
    at Sparky.emit (events.js:110:17)
    at Emitter.onevent (/opt/wifi-chat/node_modules/primus-emitter/lib/emitter.js:112:20)
    at Emitter.ondata (/opt/wifi-chat/node_modules/primus-emitter/lib/emitter.js:57:14)
    at Sparky.ondata (/opt/wifi-chat/node_modules/primus-emitter/lib/emitter.js:41:17)
    at Sparky.emit (events.js:110:17)
    at decoding (/opt/wifi-chat/node_modules/primus/spark.js:206:13)
    at Primus.decoder (/opt/wifi-chat/node_modules/primus/parsers/json.js:32:3)
    at Sparky.message (/opt/wifi-chat/node_modules/primus/spark.js:179:12)
```

**Comment by lloydwatkin** [comment]
Fixed in ab3df8a0f08da9e9d5ee6ba82224571f704be64e
